ospd/ospd.py

```diff
@@ -665,6 +673,10 @@ def sort_host_finished(
     def set_scan_progress(self, scan_id: str):
         """Calculate the target progress with the current host states
         and stores in the scan table."""
+        # Get current scan progress for debugging purposes
+        logger.debug("Calculating scan progress with the following data:")
+        self._get_scan_progress_raw(scan_id)
```

Comment:
I guess some debug output is missing in the log statement. Should `self._get_scan_progress_raw(scan_id)` get printed in the log statement?

Reply:
each time that `self._get_scan_progress_raw(scan_id) ` is called, it will log the current progress data. It is a debug message.

It looks like:
```
OSPD[12144] 2021-03-02 14:14:11,495: DEBUG: (ospd.ospd) Calculating scan progress with the following data:
OSPD[12144] 2021-03-02 14:14:11,496: DEBUG: (ospd.ospd) f2d2bb2e-1a2b-4e9c-b0f9-e1a1bedd3608: Current scan progress: 99,
OSPD[12144] 2021-03-02 14:14:11,497: DEBUG: (root) f2d2bb2e-1a2b-4e9c-b0f9-e1a1bedd3608: Current progress: 
{'count_alive': 3,
 'count_dead': 237,
 'count_excluded': 0,
 'count_total': 241,
 'current_hosts': {'192.168.0.227': 100},
 'overall': 99}
```